aurawatch.lua

```diff
@@ -560,7 +560,7 @@ function AuraWatch:COMBAT_LOG_EVENT_UNFILTERED()
 	if not interrupts[spellId] then return end
 
 	-- Apparently SPELL_INTERRUPT doesn't capture interrupts that are used on channelled abilities
-	if event ~= "SPELL_INTERRUPT" or event ~= "SPELL_CAST_SUCCESS" then return end
+	if event ~= "SPELL_INTERRUPT" and event ~= "SPELL_CAST_SUCCESS" then return end
```

Comment:
This fixes interrupt lockout timers not showing in aura watch. The old expression would always return immediately no matter what event is called.